rllib/core/learner/learner.py

```diff
@@ -278,6 +273,8 @@ def __init__(
         self._named_optimizers: Dict[str, Optimizer] = {}
         self._params: ParamDictType = {}
         self._module_optimizers: Dict[ModuleID, List[str]] = defaultdict(list)
+        # Registered metrics to be returned from `Learner.update()`.
```

Comment:
This is the new registry holding sub-dicts for each module ID.
These metrics will be "compiled" automatically into the final results in the `compile_results` method.